core/db/entities.py

```diff
@@ -397,33 +345,39 @@ class OutcomeData(BaseModel):
 
     __tablename__ = "outcome_data"
 
-    project_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("project_dim.id"), nullable=False)
+    submission_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("submission_dim.id"), nullable=False)
+    programme_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("programme_dim.id"), nullable=True)
+    project_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("project_dim.id"), nullable=True)
+    outcome_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("outcome_dim.id"), nullable=False)
+
     start_date = sqla.Column(sqla.DateTime(), nullable=False)
     end_date = sqla.Column(sqla.DateTime(), nullable=False)
-
-    # TODO: as per comment on output
     unit_of_measurement = sqla.Column(sqla.String(), nullable=False)
     geography_indicator = sqla.Column(
         sqla.Enum(const.GeographyIndicatorEnum, name="outcome_data_geography"), nullable=False
     )
     amount = sqla.Column(sqla.Float(), nullable=False)
     state = sqla.Column(sqla.Enum(const.StateEnum, name="outcome_data_state"), nullable=False)
 
```

Comment:
we also need to add a new field to this one: "higher_frequency", nullable string